### PR TITLE
PatientInfo and Images tables added.

### PR DESCRIPTION
**Pull request by FSuhov**
PatientInfo and Images tables added.

### CODE REVIEW

File: src/Schema/Schema.sqlproj [hunk 1]
```diff
@@ -62,15 +62,19 @@
...
     <Folder Include="Views" />
     <Folder Include="SchemaConversion" />
+    <Folder Include="Tables\Patient" />
   </ItemGroup>
   <ItemGroup>
     <None Include="SchemaConversion\ClientSchemaConversion.sql" />
   </ItemGroup>
   <ItemGroup>
-    <Build Include="Tables\UsersLogin.sql" />
-    <Build Include="Tables\Roles.sql" />
```
Comment (Anastasiia37) [inline src/Schema/Schema.sqlproj]: I think that this two tables need to be left in our projects

File: src/Schema/Tables/Patient/Images.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Images]
+(
+	[ImageId] INT NOT NULL  IDENTITY(1,1),
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/Images.sql]: You have extre space before IDENTITY and don`t have extra space before second 1
[ImageId] INT NOT NULL IDENTITY(1, 1),

File: src/Schema/Tables/Patient/Images.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Images]
+(
+	[ImageId] INT NOT NULL  IDENTITY(1,1), 
+    [ImageName] VARCHAR(100) NULL, 
+    [Image] VARBINARY(MAX) NULL
+	CONSTRAINT PK_Images PRIMARY KEY CLUSTERED ( [ImageId] )
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/Images.sql:7]: Before CONSTRAINT we should have one empty line

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Genre] NCHAR(10) NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL
+	CONSTRAINT PK_Patient PRIMARY KEY CLUSTERED ( [PatientId] )
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)    
+	CONSTRAINT FK_Images_Id FOREIGN KEY ( [ImageId] ) REFERENCES dbo.Images( [ImageId] )
+
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: Delete this line

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 5]
```diff
@@ -0,0 +1,18 @@
+﻿CREATE TABLE [dbo].[PatientInfo]
+(
+	[PatientId] INT NOT NULL  IDENTITY(1,1),
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: In our schema in this table we named primary key as [Id]. And you decide to name it [PatientId]. We have many foreign keys with this column and we don`t know about your decision. For example, I wrote 
CONSTRAINT FK_PatientBills_PatientInfo FOREIGN KEY ( [PatientId] ) REFERENCES PatientInfo( [Id] )
and when we put our tables together, our database will not work. It`s not good.

Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: The same about spaces

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    [LastName] VARCHAR(50) NOT NULL, 
+    [Country] VARCHAR(50) NULL, 
+    [City] VARCHAR(50) NULL, 
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Genre] NCHAR(10) NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL
+	CONSTRAINT PK_Patient PRIMARY KEY CLUSTERED ( [PatientId] )
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: Naming:
CONSTRAINT PK_PatientInfo

Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: One empty line before CONSTRAINT

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: Use [ColumnName] for column names

Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: And maybe more spaces?
CONSTRAINT UC_Person UNIQUE ( [FirstName], [LastName], [BirthDate] )

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    [City] VARCHAR(50) NULL, 
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Genre] NCHAR(10) NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL
+	CONSTRAINT PK_Patient PRIMARY KEY CLUSTERED ( [PatientId] )
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)    
+	CONSTRAINT FK_Images_Id FOREIGN KEY ( [ImageId] ) REFERENCES dbo.Images( [ImageId] )
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: Naming 
CONSTRAINT FK_Table1_Table2

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+(
+	[PatientId] INT NOT NULL  IDENTITY(1,1), 
+    [FirstName] VARCHAR(50) NOT NULL, 
+    [LastName] VARCHAR(50) NOT NULL, 
+    [Country] VARCHAR(50) NULL, 
+    [City] VARCHAR(50) NULL, 
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Genre] NCHAR(10) NULL,
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: I think it`s not good. We can write anything here

**Review by Anastasiia37 — COMMENTED** [review]
Also you don`t have scrypt with static data

File: src/Schema/Tables/Patient/Images.sql [hunk 10]
```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[Images]
+(
+	[ImageId] INT NOT NULL IDENTITY(1, 1), 
+    [ImageName] VARCHAR(100) NULL, 
+    [Image] VARBINARY(MAX) NULL
+
+	CONSTRAINT PK_Images PRIMARY KEY CLUSTERED ([ImageId])
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/Images.sql]: More spaces:
CONSTRAINT PK_Images PRIMARY KEY CLUSTERED ( [ImageId] )

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+    [City] VARCHAR(50) NULL, 
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Gender] VARCHAR(6) NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL
+
+	CONSTRAINT PK_PatientInfo PRIMARY KEY CLUSTERED ( [PatientId] )
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: CONSTRAINT UC_Person UNIQUE ( [FirstName], [LastName], [BirthDate] )

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+    [Address] VARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Gender] VARCHAR(6) NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL
+
+	CONSTRAINT PK_PatientInfo PRIMARY KEY CLUSTERED ( [PatientId] )
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)    
+	CONSTRAINT FK_Images_Id FOREIGN KEY ([ImageId]) REFERENCES dbo.Images([ImageId])
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: CONSTRAINT FK_PatientInfo_Images FOREIGN KEY ( [ImageId] ) REFERENCES dbo.Images( [ImageId] )

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+	CONSTRAINT GenderEnum CHECK (Gender IN ('Male', 'Female'))
```
Comment (Anastasiia37) [inline src/Schema/Tables/Patient/PatientInfo.sql]: CONSTRAINT GenderEnum CHECK ( [Gender] IN ('Male', 'Female') )

**Review by Anastasiia37 — APPROVED** [review]

File: src/Schema/Tables/Patient/PatientInfo.sql (hunk 5 above)
Comment (smakitech) [reply src/Schema/Tables/Patient/PatientInfo.sql]: Yes, it's serious issue, but I think the meaningful naming for primary keys it's a good practice. We haven't merged our branches yet, so I think Alexandr could gently announce this point on  meeting, and all of the team members could change reference to PatientInfo before merging.

File: src/Schema/StaticData/static_UserLogin.sql [hunk 14]
```diff
@@ -25,4 +25,34 @@ IF ( NOT EXISTS(SELECT *
...
+  END
+
+GO
+
+/* 
+ Post-Deployment Script For inserting Test Patient.
+ This file contains SQL statements that will be appended to the build script.
+*/
+
+IF (NOT EXISTS(SELECT *
```
Comment (anton-borodulin-softserve) [inline src/Schema/StaticData/static_UserLogin.sql]: @FSuhov  I created PR to have a possibility have splitted static files. You can merge my branch to yours and make a separate file for patient test data.

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+	[PatientId] INT NOT NULL IDENTITY(1, 1), 
+	[FirstName] NVARCHAR(50) NOT NULL, 
+    [LastName] NVARCHAR(50) NOT NULL, 
+    [Country] NVARCHAR(50) NULL, 
+    [City] NVARCHAR(50) NULL, 
+    [Address] NVARCHAR(MAX) NULL, 
+    [BirthDate] DATE NOT NULL, 
+    [Phone] VARCHAR(13) NULL, 
+    [Gender] BIT NULL,
+	[Email] VARCHAR(50) NULL,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql:12]: Please format query to have the same spaces on each line.

Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql:12]: I saw that Julia decided to have phone number 12 characters long. You need to have the same design over the database.

According to WIKI numbers can be up to 15 digits. But make sure if you want some more readable formats like
'+38-067-378-37-43' - only such Ukrainian number is 17 characters long. Or you will stick to '+380673783743'
Think about what is the best solution.

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+	[ImageId] INT NULL,
+	[IsDeleted] BIT NOT NULL DEFAULT 0
+
+	CONSTRAINT PK_PatientInfo PRIMARY KEY CLUSTERED ( [PatientId] )
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql:16]: here you have '[]' over column

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+	CONSTRAINT UC_Person UNIQUE(FirstName, LastName, BirthDate)
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql]: here you don't have '[]'

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+	CONSTRAINT UC_Email UNIQUE(email)
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql]: here column is lower case, one line up it was InitCapital

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+	CONSTRAINT FK_Images_Id FOREIGN KEY ([ImageId]) REFERENCES dbo.Images([ImageId])
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql]: for [ImageId] you have '[]' but '[dbo].[Images]' without '[]'. You need to have the same syntax over DB.

**Review by anton-borodulin-softserve — COMMENTED** [review]
Please fix code formatting.

File: src/Schema/Tables/Patient/PatientInfo.sql [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+	[Phone] VARCHAR(13) NULL, 
+	[Gender] TINYINT NULL,
+	[Email] VARCHAR(50) NULL,
+	[ImageId] INT NULL,
+	[IsDeleted] BIT NOT NULL DEFAULT 0
+
+	CONSTRAINT PK_PatientInfo PRIMARY KEY CLUSTERED ( [PatientId] )
+	CONSTRAINT UC_Person UNIQUE( [FirstName], [LastName], [BirthDate] )
+	CONSTRAINT UC_Email UNIQUE( [Email] )
+	CONSTRAINT FK_Images_Id FOREIGN KEY ( [ImageId] ) REFERENCES dbo.Images( [ImageId] )
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Patient/PatientInfo.sql]: @FSuhov  [dbo].[Images]

**Pull request merged**